crates/solidity/testing/perf/npm/src/benchmarks/subjects/slang.subject.mts

```diff
@@ -61,11 +61,7 @@ export class SlangSubject implements Subject {
         }
 
         if (!(definition || reference)) {
-          const ancestor = cursor.ancestors().next();
-          // Ignore experimental pragma's identifiers
-          if (!ancestor || ancestor.kind != NonterminalKind.ExperimentalFeature) {
-            neitherDefNorRefs.push(cursor.node.unparse());
-          }
+          neitherDefNorRefs.push(cursor.node.unparse());
```

Comment:
Seems like there are still a few instances of `ExperimentalFeature` checks that should be removed:

- crates/solidity/outputs/cargo/tests/src/binder/renderer.rs
- crates/solidity/outputs/cargo/tests/src/bindings/bindings_output/renderer.rs